Examples/PullServerDeployementVerificationTest/PullServerSetupTests.ps1

```diff
@@ -0,0 +1,135 @@
+﻿
```

Comment:
I will create a separate issue to make these tests part of the integration test for 'xwebDSCService" resource.